documentation/docs/getting_started/rust.mdx

```diff
@@ -8,79 +8,105 @@ keywords:
 - system environment variables
 - getting started
 ---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
 # Getting Started with Rust
 
+The wallet.rs library is published on [crates.io](https://crates.io/crates/iota-wallet/versions). You can also download the source code from the [official GitHub repository](https://github.com/iotaledger/wallet.rs).
```

Comment:
We only started publishing the stardust versions, because in production we still have not published dependencies which make it impossible for us to publish to crates.io

Reply:
```suggestion
You download the source code for the wallet.rs library from the [official GitHub repository](https://github.com/iotaledger/wallet.rs).
```
Sorry about that, I got confused with the 0.1 version that's published. I think this ought to fix it.  Thanks for the clarification :)